tests/grid/grid_in_vtk_3d_03.vtk

```diff
@@ -34,9 +34,9 @@ CELL_TYPES 4
 12
 12
 CELL_DATA 4
-SCALARS MaterialID double 1
+SCALARS MaterialID int 1
```

Comment:
I would be more comfortable if there were some kind of assertion that checks that `int` is indeed what this table uses. If you ignore the type specified here, and some input file uses `double`, then your parsing isn't going to succeed.

Reply:
We ignore the data type in current line 391 of the .cc file.